### ISSUE TITLE
I dont know if this repo can support Laravel?

### ISSUE THREAD

**Issue by SharkIng** [acim/phpFreeRad#1]

**Comment by acim** [comment]
Yes, this is general library, but unfortunately it is far from finished. I've lost interest in further development as it looked like no one is interested in. If I decide to continue the development, I will probably use Propel 2 beneath. I actually have the whole system related to ISP but it is not open source.
